modules/special-rules/pages/a-dying-race.adoc

```diff
@@ -0,0 +1,16 @@
+= A Dying Race
+
+Every Eldar casualty is a tragedy, one step closer to the race's extinction.
+
+---
+
+Apply these rules to detachments with the {a-dying-race} ability:
+
+* If the detachment is wiped out, immediately remove D3 points from the player's army morale. [either/or]
```

Comment:
```suggestion
* If the detachment is destroyed, remove D3 points from your army morale.
```
"Wiped out" is a sort of colloquialism, so I think plain English 'destroyed' is better. Arguably, "If every unit in the detachment is destroyed..." might be better still, but I'm not sure.
I don't think we need the adverb "immediately" when we use this imperative form. Even more so if we were to replace "When..." with "If..." — though that does sort of imply that your Eldar **will** cop it... >-D

Reply:
'Destroyed' is a specific type of result on the war engine Catastrophic Damage Charts, and is the result that gives the enemy half the WE's morale value as boost for seeing it blow up ([Compendium link, bullet 2](https://thehobby.zone/resources/e40k-compendium/Content/Battles/ScenarioSpecialRules/ArmyMorale.htm#link2)). So I don't think it's a safe word to use here. GW used 'wiped out' relatively frequently.

I did have a 'when the last unit in the detachment is killed' option, but it looked messy.

Some form of time reference is required, because the player's army morale drops as soon as they lose the detachment, while almost all other changes to army morale happen in the Rally phase.

Reply:
Well, "GW uses it a lot" isn't exactly a high bar for writing standards, as we know. 'Wiped out' is an idiom (not colloquialism). Idioms (and colloquialisms) are okay and perhaps interesting for creative writing, but very bad practice in rules text.

"Destroyed" is in use for much more than just the Catastrophic Damage charts. Apart from general usage, in the rules usage there is frequent mention of units being destroyed. I think we need to be explicit about when we mean the specific damage chart result (and when 'Wrecked' does or doesn't count*) — not avoid the term for any other usage. So I strongly disagree that 'destroyed' is a problematic term. Even more so in my alternate construction "If every unit in the detachment is destroyed..."

_*Though we should be mindful of 'progressive disclosure' too, so as not to overwhelm newbies with advanced rules stuff._

Yeah, options like 'when the last unit in the detachment is killed' occurred to me too, but I do try to avoid that sort of thing — the more so as it is passive case. If we were forced to use a construction like that somewhere then "When you are forced to remove the last unit as a casualty" would be appropriate and consistent.

I see your point about the time reference given the exception from usual timing. As mentioned, 'when' instead of 'if' would work, but as I say, unfortunately that sorts of implies an inevitability...
Adverbs are frowned upon but hard to avoid sometimes, so rather than tie ourselves up in knots, let's keep 'immediately' in this case.

Reply:
More properly, while we're at it:
```suggestion
* If the detachment is destroyed, immediately subtract D3 points from your army morale.
```

I can't agree with your idea that figurative text can be more clear than plain English. Idioms, colloquialisms, figurative terms -- these all demand that the reader make some sort of connection that plain English doesn't, and assume that the reader has English as their first language... and even that the reader has the same cultural background.

These things can be useful or interesting for persuasive copy (marketing, targeting a specific audience), or entertaining copy stories and such - but they're bad practice for rules and other instructional or reference content.

Reply:
We'll have to agree to disagree with this specific example, so use what you want and I'll grit my teeth about it looking 'ugly'.

Regarding languages, I work with wargames rules in Spanish and English, and sometimes French and German, and spend a lot of time discussing rules with people whose first language (and sometimes second language) isn't English. Raise/lower are commonly used and understood terms, especially in the context of morale.

Reply:
Suggestion added to commit batch.